src/scss/style.scss

```diff
@@ -1,14 +1,103 @@
+$font: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
+$backgroundColor: #fff;
+$secondaryColor: #f08080;
+
 body {
-  background-color: pink;
-  font-family: 'Trebuchet MS', 'Lucida Sans Unicode', 'Lucida Grande', 'Lucida Sans', Arial, sans-serif;
-  height: 2000px;
+  background-color: $backgroundColor;
+  font-family: $font;
+  margin: 0;
+}
+
+img {
+  width: 100%;
+  height: auto;
 }
 
-.content {
-  position: absolute;
-  top: 600px;
-  background-color: #fefefe;
-  padding: 50px;
-  margin: 20px;
-  box-shadow: inset 1px 1px 11px 0px black;
+* {
+  box-sizing: border-box;
 }
+
+.card {
+  background-color: #eee;
```

Comment:
You already started using variables for colour. Please continue doing it here as well.